.github/workflows/node.js.yml

```
@@ -0,0 +1,33 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    env:
+      working_dir: "exercises/webapp"
+
+    defaults:
+      run:
+        working-directory: ${{env.working_dir}}
+
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
```

Good Job! Just a small optional addition from me: you can split all run commands into one line, smth like:
```suggestion
      - run: |
               npm install
               npm run lint
```